exercises/two-fer/two-fer.js

```diff
@@ -0,0 +1,8 @@
+//
+// This is only a SKELETON file for the 'Two fer' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+
+export const twofer = () => {
```

Comment:
```suggestion
export const twoFer = () => {
```

The casing is incorrect. 

@tejasbubane maybe we want to use `export function twoFer` instead? Given that this is the first exercise and all?? What do you think?

Reply:
I prefer `twoFer` given the JS convention of camelCase names & exercise name being `two-fer`.

Reply:
I actually meant `export const twoFer` vs `export function twoFer`. What are your thoughts on that @tejasbubane ?

Reply:
I think `const` will be fine.